src/main/java/roomescape/domain/member/api/login/ApiLoginController.java

```diff
@@ -0,0 +1,56 @@
+package roomescape.domain.member.api.login;
+
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseCookie;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import roomescape.domain.member.domain.Member;
+import roomescape.domain.member.error.exception.MemberErrorCode;
+import roomescape.domain.member.error.exception.MemberException;
+import roomescape.domain.member.service.MemberService;
+import roomescape.domain.member.service.dto.MemberLoginRequest;
+import roomescape.domain.member.service.dto.MemberResponse;
+
+@RestController
+@RequestMapping("/login")
+public class ApiLoginController {
+
+    private static final String TOKEN = "token";
+
+    public ApiLoginController(MemberService memberService) {
+        this.memberService = memberService;
+    }
+
+    private final MemberService memberService;
+
+
+    @PostMapping
+    public ResponseEntity<?> login(@RequestBody MemberLoginRequest memberLoginRequest) {
+        String token = memberService.login(memberLoginRequest);
+        ResponseCookie responseCookie = ResponseCookie.from(TOKEN, token)
+                .httpOnly(true)
+                .path("/")
+                .maxAge(60)
+                .build();
+        return ResponseEntity.ok().header(HttpHeaders.SET_COOKIE, responseCookie.toString()).build();
+    }
+
+    @GetMapping("/check")
+    public ResponseEntity<?> loginCheck(HttpServletRequest httpServletRequest) {
```

Comment:
```suggestion
    public ResponseEntity<MemberResponse> loginCheck(HttpServletRequest httpServletRequest) {
```

응답이 명확하므로 와일드카드를 사용할 필요는 없어보이네요.